src/components/modal/Alert.tsx

```diff
@@ -0,0 +1,69 @@
+import Icon from "@/components/icon";
+import { useResponsive } from "@/hooks/use-responsive";
+import clsx from "clsx";
+import { motion } from "motion/react";
+import { ReactNode } from "react";
+import Modal from "./Modal";
+
+interface Props {
+  isOpen: boolean;
+  onClose?: () => void;
+  onExit?: () => void;
+  header?: ReactNode;
+  title: string;
+  message?: string;
+  content?: ReactNode;
+  actions?: ReactNode[];
+}
+
+export default function Alert({
+  isOpen,
+  onClose,
+  onExit,
+  header,
+  title,
+  message,
+  content,
+  actions = [],
+}: Props) {
+  const { isMobile } = useResponsive();
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} onExit={onExit}>
+      <motion.div
+        className={clsx(
+          "bg-background-primary px-4 pt-4 pb-8",
+          isMobile && "rounded-t-3xl",
+          isMobile || "w-[384px] rounded-3xl"
```

Comment:
taliwind.css에서 breakpoint를 커스텀해서 사용할 수 있는데요,
클래스네임에서 아래와 같이 적용할 수 있습니다.
참고용으로만 봐주시면 좋을 것 같습니다.😊
```
// tailwind.css
@theme {
  --breakpoint-mobile: 375px;
  --breakpoint-tablet: 745px;
  --breakpoint-desktop: 1200px;
}
```

```suggestion
          "rounded-t-3xl tablet:rounded-3xl tablet:w-[384px]"
```

Reply:
제안해 주신 방법으로 `Alert`과 `Sheet`를 수정했습니다.